deploy/olm-catalog/kogito-operator/kogito-operator.package.yaml

```
@@ -1,7 +1,7 @@
 channels:
-- currentCSV: kogito-operator.v0.10.0
+- currentCSV: kogito-operator.v0.11.0
   name: alpha
-- currentCSV: kogito-operator.v0.10.0
+- currentCSV: kogito-operator.v0.11.0
```

```suggestion
- currentCSV: kogito-operator.v0.9.1
```
We are not going to update the dev-preview channel yet